doc/UserGuide.md

```diff
@@ -41,6 +41,8 @@ Format: `list`
 ## Finding all persons containing any keyword in their name: `find`
 Finds persons whose names contain any of the given keywords.<br>
 Format: `find KEYWORD [MORE_KEYWORDS]`
+Finds person with their recorded phone number. <br>
```

Comment:
Editing line 42 rather than writing this line out will make your function description clearer